helm/robusta/values.yaml

```diff
@@ -1,6 +1,8 @@
 # most of this file is documented at https://docs.robusta.dev/master/user-guide/configuration.html
 # if you have any questions, feel free to ask via github issues or Slack (see link on robusta.dev)
 
+# NOTE: Do not use this file to install Robusta. See the installation instructions instead https://docs.robusta.dev/master/installation.html
```

Comment:
I would still explain why you shouldn't use this file directly. E.g. 

`Do not use this file as-is when installing Robusta. It has missing values that need to be filled in. See the installation...`

Reply:
Thank you, fixed it.